README.md

```diff
@@ -1,115 +1,101 @@
-<div align="center">
-  <a href="https://github.com/webpack/webpack">
-    <img width="200" height="200" src="https://webpack.js.org/assets/icon-square-big.svg">
-  </a>
-</div>
-
 [![npm][npm]][npm-url]
 [![node][node]][node-url]
 [![deps][deps]][deps-url]
-[![tests][tests]][tests-url]
+[![test][test]][test-url]
 [![coverage][cover]][cover-url]
 [![chat][chat]][chat-url]
 
-# webpack-log
-
-A common logging module for the Webpack ecosystem. `webpack-log` leverages
-[`loglevelnext`][loglevelnext].
+<div align="center">
+  <img width="160" height="160" src="docs/logo.svg">
+  <a href="https://github.com/webpack/webpack">
+    <img width="200" height="200" src="https://webpack.js.org/assets/icon-square-big.svg">
+  </a>
+  <h1>webpack Log</h1>
+  <p>A common logging module for the webpack ecosystem</p>
+</div>
 
-## Getting Started
 
-First thing's first, install the module:
+<h2 align="center"></h2>
 
-```console
-npm install webpack-log --save
+```bash
+npm i -D webpack-log
 ```
 
-_Note: We do not recommend installing this module globally._
-
-## Requirements
+> ⚠️ We do not recommend installing this module globally
 
-`webpack-log` requires Node v6 or higher.
-
-## Usage
+<h2 align="center"></h2>
 
 ```js
-const weblog = require('webpack-log');
-const log = weblog({ name: 'wds' }) // webpack-dev-server
+const log = require('webpack-log')
+const logger = log({ name: 'wds' })
 
-log.info('Server Starting');
+logger.info('Server Starting')
 ```
 
-The code above will produce:
+![output](docs/output.png)
+
+> ℹ️ The logger returned is unique by default, due to the nature of the `webpack` ecosystem. Please reference the [`unique`](#unique) option below for disabling this feature and to **force caching**
 
-![output](output.png)
+<h2 align="center">Options</h2>
```

Comment:
Let's don't use `html` in markdown for headers, it is very bad, just use `## Options`